spec/replicate_spec.cr

```diff
@@ -46,5 +47,10 @@ describe Volgen do
       volfile = Volgen.generate(SHD_TMPL, DIST_REP3_VOLDATA_JSON)
       volfile.should eq File.read("spec/sample_output/shd_dist_replica3.vol")
     end
+
+    it "checks if volfile rendered for Arbiter volume" do
+      volfile = Volgen.generate(REPLICA_TMPL, ARBITER_VOLDATA_JSON)
```

Comment:
storage unit data is passed here.

Reply:
Ack. Used Replica 3, Arbiter 1 Volume Data